notifications_utils/formatters.py

```diff
@@ -27,8 +27,9 @@ def unlink_govuk_escaped(message):
 
 def linkify(text):
     return re.sub(
-        #             url is anything without whitespace, >, or <
-        r'(https?:\/\/[^\s\<\>]+)($|\s)',
+        #  url is anything without whitespace, >, or <
+        #  Can’t be preceded by ( because that’s probably a Markdown link
+        r'(?<!\()(https?:\/\/[^\s\<\>]+)($|\s)',
```

Comment:
quick thing - i think we should specifically limit this to `](`, because it seemsthat people have been using brackets round links in text anyway, eg